IsraelHiking.Web/src/application/services/ihm-title.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from "@angular/core";
+import { ResourcesService } from "./resources.service";
+import { Title } from "@angular/platform-browser";
+
+@Injectable()
+export class IHMTitleService {
```

Comment:
Remove this class and use the title, if we'll need to use it somewhere else then we should consider creating a small wrapper to reduce duplication of code. but currently there's only one place that uses this service...

Reply:
This is currently used from `main-map.component.ts` and `public-poi-sidebar.component.ts`.

How would you like to nicely reduce duplication without a service?

Reply:
In main-map you only use the clear which I'm not sure it's needed...?
But if it's needed, then yes, this service is needed.
I would change the method from `clear` to `useDefault` for clarity.